src/components/Topics.tsx

```diff
@@ -33,24 +34,24 @@ const Topics = ({
       className={`flex flex-col rounded-lg dark:bg-dark-dark bg-light-dark my-2 p-2`}
     >
       <p className="text-xl p-2">{title}</p>
-      <div className="flex flex-row flex-wrap justify-around">
-        {topics.map((topic) => {
+      <div className="flex flex-row flex-wrap justify-around px-4">
+        {topics.map((topic, key) => {
           const selected = selectedTopics.some((t) => topic.id === t);
           return (
-            <button
-              disabled={readOnly}
-              key={topic.name}
-              className={`rounded-full m-2 ${
-                selected ? "gradient-red" : "gradient-red-faded"
-              } ${readOnly ? "cursor-default" : ""}`}
-              onClick={
-                selected
-                  ? () => removeCallback(topic)
-                  : () => addCallback(topic)
-              }
-            >
-              <span className="px-2 py-1 text-white">{topic.name}</span>
-            </button>
+            <div className="flex-inital py-2" key={key}>
```

Comment:
Just a tip, when you put key={key}, it's not really good. As in your html you will have 0,1,2,3,etc... But elsewhere in the app you can have the same. Imagine you put 2 components with the same key={key} then you'll have two times html elements with same key. 
The trick is just to put for an exemple key={`topic-${key}`} and then it's more readable, no duplicates, and happy html scroller :D